WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalBluetoothRequired.swift

```diff
@@ -48,7 +48,11 @@ final class CardPresentModalBluetoothRequired: CardPresentPaymentsModalViewModel
     }
 
     func didTapSecondaryButton(in viewController: UIViewController?) {
-        viewController?.dismiss(animated: true, completion: {[weak self] in
+        guard let viewController else {
+            return primaryAction()
```

Comment:
❓ just wanted to make sure, the dismiss isn't needed before calling `primaryAction()`?